### ISSUE TITLE
Integration testing for vest program

### ISSUE THREAD

**Issue by danpaul000** [solana-labs/solana#6234]
https://github.com/solana-labs/solana/pull/5987

**Comment by danpaul000** [comment]
Depends on #6397

**Comment by danpaul000** [comment]
@mvines is this issue still in scope for SLP1 QA if the Vest program itself is not in scope for SLP1?  Any vested/unvested token balances in the genesis block will still need to be tested before we release genesis into the world, even if the vest program itself isn't running, is that right?

**Comment by danpaul000** [comment]
Looks like we will likely not be having vest accounts (and vest program is no longer scoped for SLP1).  Closing this out.
